RocketChat/Rocket.Chat
# [FIX] UI/UX issues on Omnichannel Managers page

## Proposed changes

Omnichannel - Managers Page:
* Adding missing label at Username input
* Fixing email address sorting
* Adding "remove" confirmation at the Grid

## Issue(s)


## How to test or reproduce


Active Omnichannel, enter into Managers page and try to sort by e-mail, try to remove an manager and you'll see that "remove" confirmation won't show up.

## Screenshots
**Before**
![image](https://user-images.githubusercontent.com/2493803/97599000-4eca8600-19e6-11eb-8c20-44cb25896202.png)

**After**
![image](https://user-images.githubusercontent.com/2493803/97599904-41fa6200-19e7-11eb-9f30-b669b019d510.png)

![image](https://user-images.githubusercontent.com/2493803/97599994-59d1e600-19e7-11eb-9881-744e575566d8.png)


## Types of changes




## Checklist



## Changelog




## Further comments
